Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -140,33 +214,66 @@ namespace AZ::Render
             }
         }
 
+        // Data required for debug drawing colliders and ragdolls
+        const AZStd::unordered_set<size_t>* cachedSelectedJointIndices;
+        EMotionFX::JointSelectionRequestBus::BroadcastResult(
+            cachedSelectedJointIndices, &EMotionFX::JointSelectionRequests::FindSelectedJointIndices, instance);
+
+        Physics::CharacterPhysicsDebugDraw::NodeDebugDrawDataFunction nodeDebugDrawDataFunction =
+            [instance, cachedSelectedJointIndices](const Physics::CharacterColliderNodeConfiguration& colliderNodeConfig)
+        {
+            return GetNodeDebugDrawData(colliderNodeConfig, instance, cachedSelectedJointIndices);
+        };
+
+        Physics::CharacterPhysicsDebugDraw::JointDebugDrawDataFunction jointDebugDrawDataFunction =
```

Comment:
nit: I think this and the above call can just be `auto` (instead of `Physics::CharacterPhysicsDebugDraw::NodeDebugDrawDataFunction`) as this can be figured out at compile time and should be faster too as there might not be an allocation of an `AZStd::function`